index.html

```diff
@@ -840,19 +840,18 @@ <h4 id="How Do I Use Resistance Points?">How Do I Use Resistance Points?</h4>
          </p>
          <h5 id="Resisting Incoming Effects">Resisting Incoming Effects</h5>
          <p>
-            Throughout the game, you may sometimes be affected by a trap, attack, incoming spell, or any other effect.  
-            Often, you will have the ability to resist some or all of these effects.
-            If an effect is able to be resisted, the GM will tell you the type of resist points you can spend (FORT, REFL, or WILL) and a Difficulty Rating (DR).
-            Often, you will be able to choose how you want to resist it, for example an exposion could be dodged by spending REFL points or you can spend FORT
-            to avoid being knocked off of your feet.  
-            You may then choose to spend as many resistance points as you wish and then roll a d12 per point spent. To
-            resist the effect you will need to roll a number of successes equal to or greater than the DR.
+            Throughout the game, you will be inflicted with negative effects from traps, attacks, incoming spells, and so on. 
```

Comment:
v minor thing, by the phrasing "and so on" is very informal, and feels very out of place for a rulebook